rpa-manager/modules/ROOT/pages/processautomation-prepare.adoc

```diff
@@ -4,85 +4,88 @@ Create and document the environment for the automation of processes.
 
 == Before You Begin
 
-* Obtain the needed privileges:
+* Ask an administrator to xref:usermanagement-manage.adoc#assign-privileges-to-a-user[assign you the needed privileges]:
 +
 [cols="1,2"]
 |===
 |*Action* |*Required Privileges*
 
 |Open the *Process Automation* module
-|*Process Automation Open*
+|Process Automation Open
 
 |Create and change global variables and credentials that can be used in the test phase
-|*Global Variables Create for Test phase*, *Global Variables Edit for Test phase*
+|Global Variables Create for Test phase, Global Variables Edit for Test phase
 
 |Create and change global variables and credentials that can be used in the production phase
-|*Global Variables Create for Production phase*, *Global Variables Edit for Production phase*
+|Global Variables Create for Production phase, Global Variables Edit for Production phase
 
 |Delete global variables and credentials
-|*Global Variables Delete*
+|Global Variables Delete
 
 |Create, change, and delete process applications
-|*Application Create*, *Application Edit*, *Application Delete*
+|Application Create, Application Edit, Application Delete
 
 |===
-+
-Learn how to xref:usermanagement-manage.adoc#assign-privileges-to-a-user[Assign Privileges to a User].
-* To use AWS credentials, you need an AWS account.
-+
-Learn about https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-quickstart.html[AWS Configuration basics^]
-* Global variables and credentials can be linked to activity parameters defined in automations. The activity parameters for a process are defined in RPA Builder. Linking global variables or credentials overwrites the initial value of the linked activity parameters in a run configuration.
+* If you have access to an Amazon Web Services (AWS) account and want to create AWS credentials for xref:rpa-builder::toolbox-aws-document-processing.adoc[RPA Builder - Document Processing with AWS], follow the instructions in https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-quickstart.html[AWS configuration basics^].
+* If you have access to a Microsoft Azure account and want to create OAuth credentials for xref:rpa-builder::toolbox-mail-operations-mail-session-outlook-with-oauth.adoc[RPA Builder - Mail Session (Outlook with OAuth)], create an application for Outlook, https://learn.microsoft.com/en-us/exchange/client-developer/exchange-web-services/how-to-authenticate-an-ews-application-by-using-oauth[authenticate an EWS application by using OAuth^], and https://learn.microsoft.com/en-us/exchange/client-developer/legacy-protocols/how-to-authenticate-an-imap-pop-smtp-application-by-using-oauth[authenticate an IMAP, POP or SMTP connection using OAuth^].
```

Comment:
A comma seems to be missing in the "authenticate an IMAP, POP, or SMTP connection using OAuth" phrase.